lecture4/src/main/java/NoteContent.java

```diff
@@ -0,0 +1,7 @@
+public class NoteContent {
```

Comment:
Not necessary  class you can use just string

Reply:
We need to use json
![json](https://user-images.githubusercontent.com/49817984/61735006-b7f72e80-ad8b-11e9-8ceb-7b922490a6fc.png)